redis/commands/cluster.py

```diff
@@ -15,8 +43,15 @@
 from .helpers import list_or_args
 from .redismodules import RedisModuleCommands
 
+if TYPE_CHECKING:
+    from redis.asyncio.cluster import ClusterNode
+
+_TargetNodesT = TypeVar(
```

Comment:
Could this theoretically be in the typing module instead? Or the Cluster file rather than the cluster commands file?

Reply:
Sure, makes sense.